### PR TITLE
Remove Mr. TyDi Datasets

### PR DESCRIPTION
**Pull request by orionw**
MiRACL datasets expand Mr. TyDi datasets. It seems redundant to have the same queries with fewer relevance annotations on the same corpus.

From #354

### CODE REVIEW

File: mteb/tasks/Retrieval/kor/KoMrtydi.py
Comment (KennethEnevoldsen) [inline mteb/tasks/Retrieval/kor/KoMrtydi.py:1]: We will need to remove it from the Korean benchmark as well. It is not on the leaderboard so it should be fine.

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]
Only one change, otherwise looks good

File: mteb/tasks/Retrieval/kor/KoMrtydi.py
Comment (orionw) [reply mteb/tasks/Retrieval/kor/KoMrtydi.py:1]: I think I did that, lmk if `run_korean_mteb` wasn't the file you were referring to.

Comment (orionw) [reply mteb/tasks/Retrieval/kor/KoMrtydi.py:1]: Otherwise, I think it is updated and ready to go now.

**Pull request merged**